.ddev/docker-compose.override.yaml

```diff
@@ -0,0 +1,11 @@
+#
+# This file can be used for overriding ENV variables.
```

Comment:
Please put environment variables in the config.yaml (`web_environment`) instead of this file. Remove this file.

Reply:
I think having a separate file makes it's easier for modification.